src/agents/tracing/processors.py

```diff
@@ -96,7 +97,8 @@ def export(self, items: list[Trace | Span[Any]]) -> None:
             logger.warning("OPENAI_API_KEY is not set, skipping trace export")
             return
 
-        data = [item.export() for item in items if item.export()]
+        # include all export() results, even if they’re empty/falsy
+        data = [item.export() for item in items]
```

Comment:
Can you elaborate more on this? If there is a certain situation to resolve, we're happy to adjust this logic, but we aren't aware of anything.